prody/dynamics/plotting.py

```diff
@@ -2381,3 +2355,43 @@ def showTree_networkx(tree, node_size=20, node_color='red', node_shape='o',
         showFigure()
 
     return mpl.gca()
+
+
+def checkColors(colors, num, labels, allowNumbers=False):
+    """Check colors and process them if needed"""
+
+    from matplotlib.colors import is_color_like
+
+    colors_dict = {}
+
+    if isinstance(colors, np.ndarray):
+        colors = tuple(colors)
```

Comment:
Why does it need a numpy array check here? I believe `is_color_like` can handle numpy arrays.

Reply:
This is something we've had for a long time. Yes, you're right - it can.